eng/common/templates/jobs/build-images.yml

```diff
@@ -40,13 +40,6 @@ jobs:
   - powershell: echo "##vso[task.setvariable variable=imageBuilderBuildArgs]"
     condition: eq(variables.imageBuilderBuildArgs, '')
     displayName: Initialize Image Builder Build Args
-  - ${{ if eq(variables['System.TeamProject'], parameters.internalProjectName) }}:
-    # Add authenticated pip feed
-    - task: PipAuthenticate@1
-      displayName: 'Pip Authenticate'
-      inputs:
-        artifactFeeds: public/dotnet-public-pypi
-        onlyAddExtraIndex: false
```

Comment:
@NikolaMilosavljevic it looks like these changes were made in eng/common which means they'll be overridden by automation. Is there another extension point we can use to insert this task? Like `parameters.customInitSteps`?

Reply:
Started https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1552